src/image_manager/SkopeoDriver.cpp

```diff
@@ -230,11 +230,12 @@ void SkopeoDriver::acquireAuthFile(const common::Config::Authentication& auth, c
     rapidjson::Pointer(jsonPointer.str().c_str()).Set(authJSON, encodedCredentials.c_str());
 
     common::createFoldersIfNecessary(authFileBasePath);
-    authFilePath = authFileBasePath / "auth.json";
+    authFilePath = common::makeUniquePathWithRandomSuffix(authFileBasePath / "auth.json");
```

Comment:
Given the way `makeUniquePathWithRandomSuffix()` works, the auth file path would result in `<base path>/auth.json-<random string>`.

Do you think something like
`common::makeUniquePathWithRandomSuffix(authFileBasePath / "sarus-auth").replace_extension(.json);`
would be more elegant?

This is a pure formality, since the code would work anyways (considering the path is passed explicitly to Skopeo).